lib/ASRouter.jsm

```diff
@@ -129,6 +129,17 @@ async function chooseBranch(seed, branches) {
   return branches[await Sampling.ratioSample(seed, ratios)][0];
 }
 
+// istanbul ignore next
+function additionalParamsForTrigger(id) {
+  if (id === "openURL") {
+    return ASRouterTargeting.Environment.recentBookmarks.then(bookmarks =>
```

Comment:
It seems kind of wasteful to have to do this on every page load if the bookmarks-related message isn't in play.

I wonder if instead of modifying all usages of`openURL` for for this use case, what do you think about a new trigger (`openBookmarkedURL`?) that skips the URL check and instead compares against `recentBookmarks` whenever a page is loaded?.